blocks/library/code/index.js

```diff
@@ -38,6 +38,14 @@ registerBlockType( 'core/code', {
 				regExp: /^```$/,
 				transform: () => createBlock( 'core/code' ),
 			},
+			{
+				type: 'raw',
+				isMatch: ( node ) => (
+					node.nodeName === 'PRE' &&
+					node.children === 1 &&
```

Comment:
Since `.children` is the NodeList, should this be testing `.children.length` or `.childElementCount` ? A few other occurrences of this in the pull request.